torchmdnet/models/tensornet.py

```diff
@@ -217,6 +217,12 @@ def forward(
             mask = (edge_index[0] < 0).unsqueeze(0).expand_as(edge_index)
             zp = torch.cat((z, torch.zeros(1, device=z.device, dtype=z.dtype)), dim=0)
             # I trick the model into thinking that the masked edges pertain to the extra atom
+            if q is None:
+                q = 0
+            else:
+                batchp = torch.cat((batch, batch[-1] + 1), dim=0)
+                qp = torch.cat((q, torch.zeros(1, device=q.device, dtype=q.dtype)), dim=0)
+                q = qp[batchp][...,None,None,None]
```

Comment:
Put this above the comment so is not a lie.

Reply:
whoops, true